inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -163,6 +164,9 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::ConstantFolding>();
     // need to convert to Gather-1 until plugins do not support Gather-7
     manager.register_pass<ngraph::pass::ConvertGather7ToGather1>();
+    // to enhance operation maintenance some plugins my choose to
+    // work with a single general Gather op with batch_dims
+    manager.register_pass<ngraph::pass::ConvertGather1ToGather7, false>();
```

Comment:
does it mean that this transformation is disabled by default and plugins must explicitly register this pass?

Reply:
as far as I understood, they need just to enable it (without registering), like it was done here
https://github.com/openvinotoolkit/openvino/pull/5552/files#diff-e5c25f780993c6d5830bb707db00ae11c4380bbdeb5d066e0fd55c8fa3100783R282-R284